.gitignore

```diff
@@ -25,6 +25,9 @@ nytprof*
 # Devel::Cover output
 cover_db/
 
+# Schema file dumped for reference in development
+**/mediawords.sql
```

Comment:
I figured it could create confusion if this is being committed by different people during overlapping development, and the file isn't used anywhere at this point

Reply:
Good idea, thanks!